lib/message.rb

```diff
@@ -0,0 +1,16 @@
+require './lib/morse_translator.rb'
+require 'pry'
+
+class Message
+    attr_reader :text,
+                :translator
+
+  def initialize(text, translator)
+    @text = text
+    @translator = translator
+  end
+
+  def translation
+    translator.translate(text)
```

Comment:
When calling an instance variable from inside the class, we should use `@translator`.